war/src/main/less/abstracts/colors.less

```diff
@@ -116,11 +116,23 @@
   --table-stripped-bg-color--hover: #e8e8e8;
 
   // Link
-  --link-color: #204a87;
-  --link-visited-color: #5c3566;
-  --link-dark-color: var(--black);
+  --link-color: var(--primary-hover);
+  --link-visited-color: var(--link-color);
+  --link-color--hover: var(--link-color);
+  --link-color--active: var(--text-color);
+  --link-text-decoration: none;
+  --link-text-decoration--hover: underline;
+  --link-text-decoration--active: underline;
+  --link-font-weight: 600;
```

Comment:
@timja just a comment. I put these non-color variables in this file because I didn't know where else to put them. I'm thinking on creating a follow-up PR to rename this file to `theme.less` and include the font variables as well. WDYT?

Reply:
yeah that would make sense